src/main/java/friendlybot/command/MarkCommand.java

```diff
@@ -0,0 +1,58 @@
+package friendlybot.command;
```

Comment:
love how you use different command classes for different commands, great and easy way to update commands in the future 😄